src/Algebra/Morphism/Definitions.agda

```diff
@@ -28,19 +28,3 @@ Homomorphic₁ ⟦_⟧ ∙_ ∘_ = ∀ x → ⟦ ∙ x ⟧ ≈ (∘ ⟦ x ⟧)
 
 Homomorphic₂ : (A → B) → Op₂ A → Op₂ B → Set _
 Homomorphic₂ ⟦_⟧ _∙_ _∘_ = ∀ x y → ⟦ x ∙ y ⟧ ≈ (⟦ x ⟧ ∘ ⟦ y ⟧)
-
-------------------------------------------------------------------------
--- DEPRECATED NAMES
-------------------------------------------------------------------------
--- Please use the new names as continuing support for the old names is
--- not guaranteed.
-
```

Comment:
Maybe it's my pessimistic mindset, but i wonder why we don't permanently retain this section, empty or otherwise, in all `stdlib` modules, as it seems inevitable that as the library evolves, and our naming strategies/policies with it, that there will always eventually be entries here? ;-)

Reply:
I'd rather not, as long as we have _somewhere_ to copy-paste it from if we need to (the style guide maybe?)

Reply:
Good suggestion!